clang/include/clang/CIR/Dialect/IR/CIRTypes.h

```diff
@@ -40,13 +40,13 @@ struct StructTypeStorage;
 /// There are three possible formats for this type:
 ///
 ///  - Identified and complete structs: unique name and a known body.
-///  - Identified and incomplete structs: unique name and unkonwn body.
+///  - Identified and incomplete structs: unique name and unknown body.
 ///  - Anonymous structs: no name and a known body.
 ///
-/// Identified structs are uniqued by their name, and anonymous structs are
-/// uniqued by their body. This means that two anonymous structs with the same
+/// Identified structs are unique by their name, and anonymous structs are
+/// unique by their body. This means that two anonymous structs with the same
```

Comment:
Here `uniqued` is a verb in passive form that refers to the process of "type uniqueing".